library/src/G4CMPQPRecombinationProcess.cc

```diff
@@ -114,7 +114,7 @@ G4bool
 G4CMPQPRecombinationProcess::IsApplicable(const G4ParticleDefinition& aPD) {
   
   // Allow all phonon types, because type is changed during tracking
-  return G4VQPProcess::IsApplicable(aPD);
+  return G4CMPVQPProcess::IsApplicable(aPD);
```

Comment:
As above, this shouldn't be necessary.  Just remove the definition entirely in this subclass.